examples/fungible/tests/cross_chain.rs

```diff
@@ -19,8 +19,8 @@ use linera_sdk::{
 /// checks that the balances on each microchain are correct.
 #[tokio::test]
 async fn test_cross_chain_transfer() {
-    let initial_amount = Amount::from(20);
-    let transfer_amount = Amount::from(15);
+    let initial_amount: Amount = "20".parse().unwrap();
```

Comment:
Agreed with @christos-h that it is weird to create a string then parse it.

Reply:
I guess we could make `Amount::from_millis(u64)`, and do the same for all prefixes for which the `u64` is guaranteed not to overflow the `Amount`.

I'm not sure what's the best way to express 10.77 tokens:
```rust
amount = "10.77".parse().unwrap();
amount = Amount::from_tokens(10) + Amount::from_millis(770);
amount = Amount::from_millis(10770);
amount = Amount::from_f64(10.77); // Readable, but can lose precision!
```

Reply:
Done; but I only added `from_tokens`, `from_milli` and `from_atto` so far, because more constructors might cause more confusion rather than clarity.